rails/benchmarks/activerecord_complex.rb

```diff
@@ -0,0 +1,103 @@
+require 'bundler/setup'
+
+require 'benchmark/ips'
+require 'json'
+
+require 'rails'
+require 'action_controller/railtie'
+require 'active_record'
+require 'sqlite3'
+require 'ffaker'
+
+TIME = (ENV['BENCHMARK_TIME'] || 5).to_i
+
+ActiveRecord::Base.establish_connection(adapter: 'sqlite3', database: ':memory:')
+
+def quietly
+  silence_stream(STDOUT) do
+    silence_stream(STDERR) do
+      yield
+    end
+  end
+end
+
+def silence_stream(stream)
+  old_stream = stream.dup
+  stream.reopen(IO::NULL)
+  stream.sync = true
+  yield
+ensure
+  stream.reopen(old_stream)
+  old_stream.close
+end
```

Comment:
I tried using `ActiveRecord::Migration.verbose = false` and it works too :smile: